### ISSUE TITLE
Search more than just the currently displayed reviews

### ISSUE THREAD

**Issue by Korb** [ImJezze/steam-review-filter#2]
Currently, to search through all reviews, the user needs to scroll to the very last review, and only then use the Steam Review Filter text search. It is not obvious that otherwise, in general, the search results will not match the search through all reviews.

I propose adding the functionality of loading all reviews to the userscript. This will slow down the display of the review page, but will allow the userscript to be used not only for filtering the first reviews.

If for some reason it is decided not to integrate automatic loading, I propose at least warning the user on the review page that without manually scrolling to the last review, the filtering will be incomplete.

Steam review keyword filter 1.0 (2021-07-17)
Tampermonkey 5.3.2 (November 7, 2024)
Mozilla Firefox 134.0b3 (64-bit)

**Comment by memeyou** [comment]
I don't think so as the content has not been loaded yet.  What would be nice is automating the process of loading all the pages.  To get the job done.  'Glad this still works today  but takes forever to manually load.
